### PR TITLE
Backport RCT USART DMA Support to MK2

### PR DESCRIPTION
**Pull request by stieg**
Adds DMA support into MK2 for all USARTS where possible.  Unfortunately GPS Tx was not possible due to unresolvable channel conflicts, but such is life.

### CODE REVIEW

File: src/devices/bluetooth.c [hunk 1]
```diff
@@ -95,6 +95,8 @@ static const char * baudConfigCmdForRate(unsigned int baudRate)
                 return "AT+BAUD7";
         case 115200:
                 return "AT+BAUD8";
+        case 230400:
```
Comment (brentpicasso) [inline src/devices/bluetooth.c]: did this sneak in from something else?

Comment (stieg) [inline src/devices/bluetooth.c]: Oh yeah... I noticed that when I was trying to debug why BT wasn't operational.  I could include that as part of another change if you like.

Comment (brentpicasso) [inline src/devices/bluetooth.c]: No problem, just curious.

**Comment by brentpicasso** [comment]
:shipit:

**Comment by stieg** [comment]
Jenkins Test this please

**Comment by stieg** [comment]
Jenkins Test this please

**Pull request merged**